Libraries/acdriver/Sources/Options.cpp

```diff
@@ -32,6 +32,8 @@ parseArgument(std::vector<std::string> const &args, std::vector<std::string>::co
         return libutil::Options::Current<bool>(&_printContents, arg);
     } else if (arg == "--compile") {
         return libutil::Options::Next<std::string>(&_compile, args, it);
+    } else if (arg == "--compile-output-filename") {
```

Comment:
Could you add a comment somewhere (maybe in the options struct, like `Libraries/xcdriver/Headers/xcdriver/Options.h`) that it's an extension? Makes it easier to keep track of.